WEB/BE/utils/crypto/index.js

```diff
@@ -0,0 +1,22 @@
+const Crypto = require('crypto');
+const secretKey = process.env.ENCRYPTIONKEY;
+
+const encrypWithAES256 = ({ Text }) => {
```

Comment:
혹시 encrypt, decrypt 아닌가여?
```suggestion
const encryptWithAES256 = ({ Text }) => {
```

Reply:
앗 오타 ㅎㅎㅎ 감사합니다 !!